wgpu-types/src/instance.rs

```diff
@@ -429,6 +437,47 @@ impl NoopBackendOptions {
     }
 }
 
+#[derive(Clone, Debug, Default, Copy, PartialEq, Eq)]
+/// Selects which presentation system to use on DX12.
+pub enum Dx12PresentationSystem {
+    /// Use the DXGI presentation system.
+    #[default]
+    Dxgi,
+    /// Use the DirectComposition presentation system.
+    Dcomp,
```

Comment:
```suggestion
    DirectComposition,
```

I should have mentioned this earlier, sorry, lets use the full name here.